lib/msal-common/src/config/ClientConfiguration.ts

```diff
@@ -274,6 +275,7 @@ function buildAuthOptions(authOptions: AuthOptions): Required<AuthOptions> {
         azureCloudOptions: DEFAULT_AZURE_CLOUD_OPTIONS,
         skipAuthorityMetadataCache: false,
         instanceAware: false,
+        redirectUri: "https://localhost",
```

Comment:
We should not be using or defaulting to localhost here - suggest making redirectUri required on the AuthOptions object to avoid

Reply:
Made `redirectUri` mandatory in msal-common and update msal-browser and msal-node accordingly.